sklearn/ensemble/tests/test_bagging.py

```diff
@@ -237,6 +237,9 @@ def fit(self, X, y):
         self.training_size_ = X.shape[0]
         self.training_hash_ = joblib.hash(X)
 
+    def predict(self, X):
+        return np.ones(X.shape[0])
```

Comment:
This is not covered. Why did you need to add this ?

Reply:
Otherwise the estimator is not a proper predictor during the parameter validation.